pegasus/sites.v3/code.org/views/top_two_hoc_tutorials.haml

```diff
@@ -0,0 +1,27 @@
+.col-50
+  .tutorial-tile
+    %a{href: CDO.studio_url("/s/applab-intro/reset")}
+      %img.tutorial-tile-img{src: CDO.code_org_url("/images/fill-485x260/app-lab/applab-intro.png")}
+    .tutorial-info{style: "min-height: 200px;"}
+      %h3.tutorial-info-h Intro to App Lab Tutorial
+      .smalltext Create your own app in JavaScript using block based programming. Or take your skills to the next level with text-based programming.
```

Comment:
Really not a fan of copying & pasting the text content across multiple pages.

Reply:
fwiw, this will be remedied when we loc these pages.

Reply:
https://github.com/code-dot-org/code-dot-org/blob/17588db3c241c572818a5fc0aa9e6bb3412b745b/pegasus/sites.v3/code.org/public/educate/applab.haml#L31

Reply:
@caleybrock is going to refactor this duplication shortly.

Reply:
Applab block is now factored out.  Took some special casing to match its neighbours' heights, since MC & YouTube both have their own ideal image heights.